File: CSharp/OpenTraceability.GDST/Events/KDEs/VesselCatchInformationList.cs
using OpenTraceability.Utility.Attributes;

namespace OpenTraceability.GDST.Events.KDEs
{
    public class VesselCatchInformationList
    {
        [OpenTraceabilityArray]
        [OpenTraceabilityObject]
        [OpenTraceability(Constants.CBVMDA_NAMESPACE, "vesselCatchInformation")]
        [OpenTraceabilityJson("cbvmda:vesselCatchInformation")]
        public List<VesselCatchInformation> Vessels { get; set; } = new List<VesselCatchInformation>();
    }
}

File: Java/OpenTraceability/src/main/java/opentraceability/gdst/events/kdes/VesselCatchInformationList.java
package opentraceability.gdst.events.kdes;

import opentraceability.Constants;
import opentraceability.utility.attributes.*;

import java.util.ArrayList;
import java.util.List;

public class VesselCatchInformationList {
    @OpenTraceabilityArrayAttribute(itemType = VesselCatchInformation.class)
    @OpenTraceabilityObjectAttribute
    @OpenTraceabilityJsonAttribute(name = "cbvmda:vesselCatchInformation")
    @OpenTraceabilityAttribute(ns = Constants.CBVMDA_NAMESPACE, name = "vesselCatchInformation")
    private List<VesselCatchInformation> vessels = new ArrayList<>();

    // Add constructor, getters, and setters here
}
